CONTRIBUTING.md

```diff
@@ -94,14 +94,36 @@ open http://localhost:8000/
 
 To publish the packages to npm, you'll need to be added as an owner for the
 packages you're publishing. Use the #design-system Slack channel for more
-information.
+information. 
+
+Before Publishing:
+
+- Know that publishing is very easy once you have access to do so **Please Be Careful** 🤗
+- As of v2 you must be on Node v8+
+- name sure you are logged into NPM properly, if you have multiple logins you can use something like [npmrc]
```

Comment:
> [npmrc]() ...

Would you like to provide the npmrc variable, and the npmrc link at the bottom? (If you prefer a follow-up PR, that's cool)